### PR TITLE
Review ch5

### PR DESCRIPTION
**Pull request by luizaandrade**

### CODE REVIEW

File: chapters/5-processing.tex
Comment (kbjarkefur) [inline chapters/5-processing.tex:71]:
```suggestion
We will treat tiding data as the first step in data cleaning,
even though in practice, 
both tidying and quality monitoring should be done simultaneously as data is received.
This is because quality assurance can only be finalized using tidied data,
when it is guaranteed that each observations are uniquely identified.
```
I am not trying to change what you are saying, but I had to read your first sentence a few times before I got what you were saying. I think this gets to the point you are trying to make quicker.

File: chapters/5-processing.tex
Comment (kbjarkefur) [inline chapters/5-processing.tex:622]:
```suggestion
then at least the confidential part of the data is required 
to remain encrypted 
and only be decrypted when used during in the data analysis process.
It is never an excuse to store confidential data unencrypted
or saved in any other insecure way,
just because it is needed in the analysis process.
```
I think we are confusing things by bringing in _secure environment_. What we really are saying is that it is not an excuse to save confidential data insecurely just because your analysis scripts reads those files. Other parts of the book covers the details of whats secure or not. Does this sound good? I think this would have prevented Jim's confusion.

File: chapters/5-processing.tex
Comment (kbjarkefur) [inline chapters/5-processing.tex:328]:
```suggestion
you are receiving administrative data from a partner,
```
To make it more clear that is what we mean, otherwise it sounds as we just as well could mean survey data from a partner

**Review by kbjarkefur — CHANGES_REQUESTED** [review]

**Comment by kbjarkefur** [comment]
I like the text you added that no matter if it is survey data or admin data "the key aspects to have in mind are data completeness, consistency and distribution". However, I agree with Jim a bit that it seems as we are suggesting only survey quality methods to both survey and admin data. We seem to refer both survey and admin data when we say "Data quality assurance requires a combination of real-time data checks and back-checks or validation audits" but I think those are just survey data. But I also agree with you that we say what we need to say, but I think we should restructure it.

I think we should start by saying that for both "the key aspects to have in mind are data completeness, consistency and distribution" and then say that for survey data you have additional tools as you do the checks as data is collected, those tools are "real-time data checks and back-checks or validation audits". Then we talk about those survey specific tools. Then we end on talking on tabulations and such (which we already do) that answers "data completeness, consistency and distribution" and this applies to both. Does that make sense? It think it comes down to that we need to be clear what applies to both and what does only apply to survey data, like HFCs.

File: chapters/5-processing.tex
Comment (luizaandrade) [reply chapters/5-processing.tex:622]:
```suggestion
at least the confidential part of the data is required 
to remain encrypted 
and only be decrypted when used during in the data analysis process.
Using confidential data in the analysis process
does *not* justify storing or sharing it in an insecure way.
```

**Comment by luizaandrade** [comment]
I think real-time quality checks apply to both. But I agree that back-checks do not. Could we say something like "in the case of survey data, real-time data quality checks include back-checks and validation audits?"

File: chapters/5-processing.tex
Comment (kbjarkefur) [reply chapters/5-processing.tex:620]: Does `*not*` formatting work in LaTeX? or is this markdown formatting baked in?

**Comment by luizaandrade** [comment]
- [ ] "Run checks promptly" instead of real-time: this means as soon as you receive it, even if it's not a high-frequency transfer
- [ ] If the data collection is goin on, checks should be done in real time
- [ ] Move back checks and validation audits to Ch 4

File: chapters/5-processing.tex
Comment (kbjarkefur) [reply chapters/5-processing.tex:620]: I think the "then" on line 613 should still be there, but a professional reviewer is the person best suited to decide what is best. Was it intentional that you removed it? Apart from that I think your last version is ready to be committed.

**Comment by kbjarkefur** [comment]
I will wait for the last items until I do my final review. I made a comment in https://github.com/worldbank/dime-data-handbook/pull/550 on whether it makes sense to split up data quality checks and between two chapters. I do not think it does. 

In addition to the real time aspect, I think there are survey specific checks and those quality checks are focused on the huge source of error that comes from humans recording the answers. I think that is what back checks, validation audits and much of HFCs are focusing on. Survey data quality checks should always be done in real time when possible. An example when it is not possible is if we receive survey data collected where the field activities are already concluded.

Then when we have secondary data that is collected in real time, for example CDR data and remote sensing. I think the quality checks there are similar to quality checks done for non-real time secondary data, the only difference is that we do those checks frequently in real time. So I do not think CDR data has more in common with survey data than any other secondary data, despite it is is received in real time.

Let me know if you disagree.

**Comment by luizaandrade** [comment]
I disagree. I think most of the survey-specific content in chapter 5 is about things that should be thought of before the data is acquired, and alongside survey planning. The changes in 87d131a reflect the content that I think should be moved to ch 4. I also added more language on this in c4c24db as I wrote this comment to give examples of both survye and other data sources.

File: chapters/5-processing.tex
Comment (luizaandrade) [reply chapters/5-processing.tex:572]: I added it back. I think it's optional, and I removed it just for stylistic reasons, but I don't feel strongly about it

**Comment by mariaruth** [comment]
> I disagree. I think most of the survey-specific content in chapter 5 is about things that should be thought of before the data is acquired, and alongside survey planning. The changes in [87d131a](https://github.com/worldbank/dime-data-handbook/commit/87d131af84dc79bf6f41142cd6a8e9f7d7e8afea) reflect the content that I think should be moved to ch 4. I also added more language on this in [c4c24db](https://github.com/worldbank/dime-data-handbook/commit/c4c24db664dcfe835b3d00eb4c60becfa122d02a) as I wrote this comment to give examples of both survye and other data sources.

Luiza, I agree. From the commit, i saw two code chunks to move to ch 4, i commented to verify. let me know if there are any others and i'll work on integrating this tomorrow morning.

**Comment by kbjarkefur** [comment]
Ok, I am happy to with what you are suggesting!

**Pull request merged**
